webpack/stories/docs/api-middleware.stories.mdx

```diff
@@ -95,12 +102,12 @@ In case you want to invoke some method or action immediately when an error occur
 we support the `handleError` param where you could pass an error handling function, e.g:
 
 dispatch(
-  get({
-    key: MY_SPECIAL_KEY,
-    url,
-    handleError: error => error.response.status === 401 ? logoutUser() : null
-    ...
-  })
+get({
+key: MY_SPECIAL_KEY,
+url,
+handleError: error => error.response.status === 401 ? logoutUser() : null
+...
+})
 );
```

Comment:
Why the indentation change?

Reply:
I think from one of the rebases.. fixed, thanks !